blogs/windows-unattended/autounattend.xml

```diff
@@ -95,7 +95,7 @@
 		</AutoLogon>
             <FirstLogonCommands>
                 <SynchronousCommand wcm:action="add">
-	            <CommandLine>powershell -noexit d:\post-install.ps1
+	            <CommandLine>powershell -noexit f:\post-install.ps1
```

Comment:
this depends on the order of the drivers you set.
if you switched between them then yes the latter would be changed